packages/atomic/cypress/integration/sort-dropdown.cypress.ts

```diff
@@ -72,6 +74,15 @@ describe('Sort Dropdown Component', () => {
     const urlHash = `sortCriteria=${encodeURIComponent(expressions[1])}`;
     cy.url().should('include', urlHash);
   });
+
+  it('should be invisible when there is no result', () => {
+    cy.get(searchbox)
+      .shadow()
+      .find('.search-input')
+      .type('lskdfjghsjdgfjsdgfhkjsdhfgkjsdgfksjdhgfkjsdgkjgdsfjh');
```

Comment:
I think this call would be a handy to have in central function, since several tests for different components will want to test the no results case. Could there already be something used by other tests?

Reply:
Easier setup would be to load the page with the q parameter set to the big incomprehensible string

Reply:
> I think this call would be a handy to have in central function, since several tests for different components will want to test the no results case. Could there already be something used by other tests?

Yes, certainly.

I'm thinking about just merging it like this however.

I'm looking at how the "setupPage" function is done for Cypress, and I'm not really happy with the way it is right now.

I'll propose a PR to help setup different scenarios, which is more like a "pipeline of stuff to configure" that you can use to inject things incrementally for a given test suite.

For example, to get a valid "no results page" you need to control the HTML + the first query parameter execution, which are both tied to the "setupPage" function.

So, to share this between test would mean just copy pasting a "setupPageNoResults", which is really not elegant.